code/game/machinery/vending.dm

```diff
@@ -821,7 +822,7 @@
 	has_logs = 1
 
 /obj/machinery/vending/cigarette
-	name = "Cigarette machine" //OCD had to be uppercase to look nice with the new formating

```

Comment:
why was this even capitalized ever

Reply:
OCD however, didn't know how to spell 'formatting' right.